pandas/tests/series/test_combine_concat.py

```diff
@@ -214,20 +214,23 @@ def test_concat_empty_series_dtypes(self):
                           Series(dtype='object')]).dtype == 'object'
 
         # sparse
+        # TODO: move?
         result = pd.concat([Series(dtype='float64').to_sparse(), Series(
             dtype='float64').to_sparse()])
-        assert result.dtype == np.float64
+        assert result.dtype == 'Sparse[float64]'
         assert result.ftype == 'float64:sparse'
 
         result = pd.concat([Series(dtype='float64').to_sparse(), Series(
             dtype='float64')])
-        assert result.dtype == np.float64
+        # TODO: release-note: concat sparse dtype
+        assert result.dtype == pd.core.sparse.dtype.SparseDtype(np.float64)
         assert result.ftype == 'float64:sparse'
 
         result = pd.concat([Series(dtype='float64').to_sparse(), Series(
             dtype='object')])
-        assert result.dtype == np.object_
-        assert result.ftype == 'object:dense'
+        # TODO: release-note: concat sparse dtype
```

Comment:
@TomAugspurger these TODOs are still present (though now in a different file).  Is this actionable?

Reply:
Probably not.